packages/slate-history/src/with-history.ts

```diff
@@ -44,16 +44,7 @@ export const withHistory = <T extends Editor>(editor: T) => {
           const inverseOps = batch.map(Operation.inverse).reverse()
 
           for (const op of inverseOps) {
-            // If the final operation is deselecting the editor, skip it. This is
```

Comment:
because "focus" (so "blur" when reversed) doesn't get stored in the history now, this if statement becomes obsolete